Packs/GoogleChatViaWebhook/Integrations/GoogleChatViaWebhook/GoogleChatViaWebhook.py

```diff
@@ -0,0 +1,251 @@
+                'ThreadName': res.get('thread').get('name'),
+                'Name': res.get('name'),
+                'SenderDisplayName': res.get('sender').get('displayName'),
+                'SenderName': res.get('sender').get('name'),
+                'SenderType': res.get('sender').get('type')
+            }
+
+            res_thread_name = res.get('thread').get('name')
+            markdown = '### Google Chat\n'
+            markdown += tableToMarkdown('Message Webhook', result)
+            results = CommandResults(
+                readable_output=markdown,
+                outputs_prefix='GoogleChatWebhook.Message',
+                outputs_key_field='name',
+                outputs=result
+            )
+
+            return results
+    except DemistoException as e:
+        return f'Error: {e}'
+
+
+def send_google_chat_custom_card_command(client: Client, blocks: dict, threadName: Optional[str]) -> CommandResults:
+    """
+    send_google_chat_custom_card command: Sends the Google Chat custom card to the provided webhook.
+
+    Args:
+        client (Client): Google Chat client to use.
+        blocks (dict): The custom card to send to the Google Chat Space (UI Kit Builder JSON blocks)
+        threadName (str): If provided, will reply to an existing thread (or create a new thread)
+
+    Returns:
+        CommandResults/dict: A ``CommandResults`` compatible to return ``return_results()``,
+        which contains the readable_output indicating the message was sent.
+    """
+    try:
+        res = client.send_google_chat_custom_card(blocks=blocks, threadName=threadName)
+
+        if res:
+            result = {
+                'SpaceName': res.get('space').get('name'),
+                'SpaceDisplayName': res.get('space').get('displayName'),
+                'SpaceType': res.get('space').get('type'),
+                'CreatedTime': res.get('createTime'),
+                'ThreadReply': res.get('threadReply', False),
+                'ThreadName': res.get('thread').get('name'),
+                'Name': res.get('name'),
+                'SenderDisplayName': res.get('sender').get('displayName'),
+                'SenderName': res.get('sender').get('name'),
+                'SenderType': res.get('sender').get('type')
+            }
+
+            res_thread_name = res.get('thread').get('name')
+            markdown = '### Google Chat\n'
+            markdown += tableToMarkdown('Custom Card Webhook', result)
+            # Add the card details to context after formatting md
+            result.update({'Cards': res.get('cardsV2')}),
+            res_thread_name = res.get('thread').get('name')
+            results = CommandResults(
+                readable_output=markdown,
+                outputs_prefix='GoogleChatWebhook.CustomCard',
+                outputs_key_field='name',
+                outputs=result
+            )
+            return results
+    except DemistoException as e:
+        return f'Error: {e}'
```

Comment:
```suggestion
    res = client.send_google_chat_custom_card(blocks=blocks, threadName=threadName)

    if res:
        result = {
            'SpaceName': res.get('space').get('name'),
            'SpaceDisplayName': res.get('space').get('displayName'),
            'SpaceType': res.get('space').get('type'),
            'CreatedTime': res.get('createTime'),
            'ThreadReply': res.get('threadReply', False),
            'ThreadName': res.get('thread').get('name'),
            'Name': res.get('name'),
            'SenderDisplayName': res.get('sender').get('displayName'),
            'SenderName': res.get('sender').get('name'),
            'SenderType': res.get('sender').get('type')
        }

        res_thread_name = res.get('thread').get('name')
        markdown = '### Google Chat\n'
        markdown += tableToMarkdown('Custom Card Webhook', result)
        # Add the card details to context after formatting md
        result.update({'Cards': res.get('cardsV2')}),
        res_thread_name = res.get('thread').get('name')
        results = CommandResults(
            readable_output=markdown,
            outputs_prefix='GoogleChatWebhook.CustomCard',
            outputs_key_field='name',
            outputs=result
        )
        return results
```
No need to add a try/except block, since if an error occurs, the try/except block in the main function will catch it.